core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -159,7 +159,7 @@
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_RESPONSE_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "/rpc/client/response_bytes",
+          "rpc client response_bytes",
```

Comment:
Have we decided this is the type of naming we want? (we need to have the general view and resource naming discussion at some point).